### ISSUE TITLE
Status of this Repo

### ISSUE THREAD

**Issue by versipellis** [jakeday/linux-surface#640]
Last commit looks to have been ~8 months ago. Should we assume that @jakeday is no longer working on this, and @qzed's fork should be what we're using?

**Comment by mmalmeida** [comment]
I am using @qzed's repository as when I got the laptop this repository seemed dead.

**Comment by archseer** [comment]
Yeah we switched to maintaining qzed's patches as a community organization: https://github.com/linux-surface/linux-surface

That way there's more than one maintainer in case one of us gets inactive.

**Comment by opensas** [comment]
If https://github.com/linux-surface/linux-surface is now the recommend fork to follow, could a warning with the link be added to this repo README.md???
I still see that this repo has way more stars than @qsed's

**Comment by stucki** [comment]
I found some bug reports in this repo / project and didn't notice the difference in the first place. It seems like many users are still reporting here although the whole project has been moved.

It would probably make sense to close all issues and leave a comment about the new repository at https://github.com/linux-surface/linux-surface/.

@jakeday Can you do that please?
